core/src/main/scala/org/scalarelational/model/ModularSupport.scala

```diff
@@ -11,19 +12,28 @@ import org.scalarelational.instruction._
  *
  * @author Matt Hicks <matt@outr.com>
  */
-trait ModularSupport extends SQLContainer with Listenable with DataTypes {
+trait ModularSupport extends SQLContainer {
+  case class Processor[T]() {
```

Comment:
Seems incredibly oversimplified. How is this better than `UnitProcessor` in powerscala? Can you prove that this is substantially faster?